frontend/components/card/CardDangerZone.vue

```diff
@@ -3,50 +3,44 @@
     class="px-5 py-5 border bg-light-act-red/30 dark:bg-dark-act-red/30 border-light-act-red dark:border-dark-act-red card-style-base text-light-text dark:text-dark-text"
   >
     <div class="flex flex-col gap-5">
-      <h3
-        class="font-bold text-left text-light-act-red dark:text-dark-act-red responsive-h3 font-display"
-      >
+      <h3 class="font-bold text-left text-light-act-red dark:text-dark-act-red responsive-h3 font-display">
         {{ $t("components.card-danger-zone.header") }}
       </h3>
-      <p class="">{{ description }}</p>
+      <p>{{ description }}</p>
       <form method="post">
-        <div
-          class="justify-between grid grid-cols-1 gap-4 sm:grid-cols-2 sm:gap-8"
-        >
+        <div class="justify-between grid grid-cols-1 gap-4 sm:grid-cols-2 sm:gap-8">
           <div class="flex flex-col gap-2">
             <label
-              for="username"
               class="font-bold text-light-text dark:text-dark-text responsive-h4"
-              :placeholder="
-                $t('components.card-danger-zone.placeholder-username')
-              "
-              >{{ $t("components.card-danger-zone.label-username") }} *</label
+              for="username"
+              :placeholder="$t('components.card-danger-zone.placeholder-username')"
             >
+              {{ $t("components.card-danger-zone.label-username") }} *
+            </label>
             <input
-              id="username"
               class="px-3 py-1 bg-transparent border rounded-md border-light-text dark:border-dark-text focus:outline-none focus:ring-1 focus:ring-black dark:focus:ring-white"
+              id="username"
             />
           </div>
           <div class="flex flex-col gap-2">
             <label
-              for="password"
               class="font-bold text-light-text dark:text-dark-text responsive-h4"
-              :placeholder="
-                $t('components.card-danger-zone.placeholder-password')
-              "
-              >{{ $t("components.card-danger-zone.label-password") }} *</label
+              for="password"
```

Comment:
This prop should likely be renamed 🤔

Reply:
Which prop do you mean that needs to be renamed?

Reply:
Nevermind on this, @luuu-xu! In the review I got a bit confused at this point and I was thinking that `for` was a prop on a component. A `for` attribute on a `<label>` is totally fine :)